src/library/Uno.Material/Styles/Controls/v2/Button.xaml

```diff
@@ -0,0 +1,798 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:um="using:Uno.Material"
+					xmlns:toolkit="using:Uno.UI.Toolkit"
+					xmlns:not_win="http://uno.ui/not_win"
+					mc:Ignorable="d not_win">
+
+	<!-- Converters -->
+	<um:FromNullToValueConverter x:Key="MaterialNullToTextButtonMarginConverter"
+								 NotNullValue="0,0,2,0"
+								 NullValue="0" />
+
+	<!-- Documented variables -->
+	<Thickness x:Key="M3MaterialButtonMinHeight">40</Thickness>
+	<Thickness x:Key="M3MaterialButtonMinWidth">48</Thickness>
+	<Thickness x:Key="M3MaterialButtonPadding">16,0</Thickness>
+	<Thickness x:Key="M3MaterialTextButtonPadding">12,0</Thickness>
+	<CornerRadius x:Key="M3MaterialButtonCornerRadius">20</CornerRadius>
+	<!-- Text button label based on Label Large Typo -->
+	<x:Double x:Key="M3MaterialButtonFontSize">14</x:Double>
+	<x:Double x:Key="M3MaterialButtonCharacterSpacing">7.143</x:Double>
+
+	<!-- ELEVATED -->
+	<Style x:Key="M3MaterialElevatedButtonStyle"
+		   TargetType="Button">
+
+		<Setter Property="Foreground"
+				Value="{StaticResource PrimaryBrush}" />
+		<Setter Property="Background"
```

Comment:
Maybe we should revisit the background colors for buttons. The officia Figma file from Google uses the "surface2" color.

There's a `surface1` to `surface5`. All of them use SurfaceColor and layer on the primary color at different opacities.
![image](https://user-images.githubusercontent.com/4793020/163509076-bf745aeb-ed36-4155-ab0b-179cd4abd188.png)